pkgs/development/python-modules/django-ninja/default.nix

```
@@ -14,16 +14,16 @@
 
 buildPythonPackage rec {
   pname = "django-ninja";
-  version = "1.3.0";
+  version = "1.3.0-unstable-20241103";
```

```suggestion
  version = "1.3.0-unstable-2024-11-13";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning